src/sentry/static/sentry/app/components/events/interfaces/stacktraceLink.tsx

```diff
@@ -74,6 +78,44 @@ class StacktraceLink extends AsyncComponent<Props, State> {
     }
   }
 
+  componentDidMount() {
+    this.promptsCheck();
+  }
+
+  async promptsCheck() {
+    const {organization} = this.props;
+
+    const prompt = await promptsCheck(this.api, {
+      organizationId: organization.id,
+      projectId: this.project?.id,
+      feature: 'stacktrace_link',
+    });
+
+    this.setState({
+      isDismissed: !prompt?.dismissedTime ? false : true,
```

Comment:
@MeredithAnya I would recommend using the `promptIsDismissed` function which will make it easier to add snooze functionality later: https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/utils/promptIsDismissed.tsx#L5